src/core/core.controller.js

```diff
@@ -416,9 +416,8 @@ module.exports = function(Chart) {
 			// after update.
 			me.tooltip.initialize();
 
-			// Last active contains items that were previously in the tooltip.
-			// When we reset the tooltip, we need to clear it
-			me.lastActive = [];
```

Comment:
We shouldn't remove this lines because me.lastActive stores active elements for hovering rather than those for the tooltip. I think the comment is imprecise.